### PR TITLE
Add the `osu!catch World Cup 2024: Quarterfinals Recap` news post

### PR DESCRIPTION
**Pull request by Realmaaas**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

**Comment by 0x84f** [comment]
[preview](https://osu2.roanh.dev/home/news/2024-06-12-osu-catch-world-cup-2024-quarterfinals-recap)

**Review by peppy — APPROVED** [review]

**Review by Walavouchey — COMMENTED** [review]
waiting on potentially shortening the lengthy sentences and descriptions, as mentioned on discord

File: news/2024/2024-06-12-osu-catch-world-cup-2024-quarterfinals-recap.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+Both teams did not struggle with the difficulty of the pool, making this match about consistency over a skill advantage, which is why a solid pick strategy was important for both teams. Unfortunately Argentina made the first misstep of the match on their fourth pick — they went for [HR3](https://osu.ppy.sh/beatmapsets/2191179#fruits/4634727), one of the easier maps in the pool, but Poland prevailed, with an FC from ::{ flag=PL }:: [trig0n](https://osu.ppy.sh/users/3704228), the only one in the lobby and his 4th FC in a row at this point in the match (he would even end up extending that streak to 6 in a row). Poland had now secured the very sought-after breakpoint they needed to win the match, but still needed to win their next two picks. [HD2](https://osu.ppy.sh/beatmapsets/2191165#fruits/4634690) and [NM1](https://osu.ppy.sh/beatmapsets/2191148#fruits/4634642) were picked and secured for respectively Poalnd and Argentina, leaving it to a 5-4 scoreline for Poland.
```
Comment (Walavouchey) [inline news/2024/2024-06-12-osu-catch-world-cup-2024-quarterfinals-recap.md:94]:
```suggestion
Both teams did not struggle with the difficulty of the pool, making this match about consistency over a skill advantage, which is why a solid pick strategy was important for both teams. Unfortunately Argentina made the first misstep of the match on their fourth pick — they went for [HR3](https://osu.ppy.sh/beatmapsets/2191179#fruits/4634727), one of the easier maps in the pool, but Poland prevailed, with an FC from ::{ flag=PL }:: [trig0n](https://osu.ppy.sh/users/3704228), the only one in the lobby and his 4th FC in a row at this point in the match (he would even end up extending that streak to 6 in a row). Poland had now secured the very sought-after breakpoint they needed to win the match, but still needed to win their next two picks. [HD2](https://osu.ppy.sh/beatmapsets/2191165#fruits/4634690) and [NM1](https://osu.ppy.sh/beatmapsets/2191148#fruits/4634642) were picked and secured for respectively Poland and Argentina, leaving it to a 5–4 scoreline for Poland.
```

File: news/2024/2024-06-12-osu-catch-world-cup-2024-quarterfinals-recap.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+It wasn't an issue for Japan though, who seemed to have the edge on Hidden, and would win their next two picks once again, on [HD3](https://osu.ppy.sh/beatmapsets/2191180#fruits/4634728) and [HD1](https://osu.ppy.sh/beatmapsets/2191164#fruits/4634684). The United Kingdom would go for a [MM2](https://osu.ppy.sh/beatmapsets/2191260#fruits/4634910) pick which worked, with a beautiful No Mod FC from ::{ flag=GB }:: [Matt_H](https://osu.ppy.sh/users/4067115), to even out the scoreline at a 4-5 on their pick. They deemed [NM1](https://osu.ppy.sh/beatmapsets/2191148#fruits/4634642) to be their best option left. With nerves getting into play, backs against the wall, the United Kingdom unfortunately couldn't beat Japan on the consistency pick and ended up losing the match 6–4, with another impressive performance for the Japanese captain and veteran, [Molqus](https://osu.ppy.sh/users/1927193), FCing once again.
```
Comment (Walavouchey) [inline news/2024/2024-06-12-osu-catch-world-cup-2024-quarterfinals-recap.md:126]:
```suggestion
It wasn't an issue for Japan though, who seemed to have the edge on Hidden, and would win their next two picks once again, on [HD3](https://osu.ppy.sh/beatmapsets/2191180#fruits/4634728) and [HD1](https://osu.ppy.sh/beatmapsets/2191164#fruits/4634684). The United Kingdom would go for a [MM2](https://osu.ppy.sh/beatmapsets/2191260#fruits/4634910) pick which worked, with a beautiful No Mod FC from ::{ flag=GB }:: [Matt_H](https://osu.ppy.sh/users/4067115), to even out the scoreline at a 4–5 on their pick. They deemed [NM1](https://osu.ppy.sh/beatmapsets/2191148#fruits/4634642) to be their best option left. With nerves getting into play, backs against the wall, the United Kingdom unfortunately couldn't beat Japan on the consistency pick and ended up losing the match 6–4, with another impressive performance for the Japanese captain and veteran, [Molqus](https://osu.ppy.sh/users/1927193), FCing once again.
```

**Review by Walavouchey — DISMISSED** [review]

**Review by Walavouchey — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**